src/SFA.DAS.EmployerIncentives.Database/Schemas/Incentives.sql

```diff
@@ -1 +1 @@
-﻿CREATE SCHEMA [incentives]
\ No newline at end of file
+﻿CREATE SCHEMA [incentives]
```

Comment:
Should this script check to see if the schema already exists, or does the DACPAC take care of that?

Reply:
yes, DACPAC does take care of it, I guess it's similar to CREATE TABLE 🤷‍♂️